website/site/content/docs/add-to-your-site.md

```diff
@@ -200,11 +200,11 @@ Let's break that down:
 
 As described above, the `widget` property specifies a built-in or custom UI widget for a given field. When a content editor enters a value into a widget, that value will be saved in the document front matter as the value for the `name` specified for that field. A full listing of available widgets can be found in the [Widgets doc](https://www.netlifycms.org/docs/widgets).
 
-Based on this example, you can go through the post types in your site and add the appropriate settings to your Netlify CMS `config.yml` file. Each post type should be listed as a separate node under the `collections` field.
+Based on this example, you can go through the post types in your site and add the appropriate settings to your Netlify CMS `config.yml` file. Each post type should be listed as a separate node under the `collections` field. See [collection types](https://www.netlifycms.org/docs/configuration-options/#collections) for more configuration options.
```

Comment:
We should change the words in the link now that the target is different. I suggest, "See the [Collections reference doc] for more configuration options."